src/function-configuration.ts

```diff
@@ -147,6 +149,11 @@ export interface RuntimeOptions {
    * Ingress settings
    */
   ingressSettings?: typeof INGRESS_SETTINGS_OPTIONS[number];
+
+  /**
+   *
```

Comment:
missing documentation?

Reply:
Good catch. I also went through and cleaned up some of these other, less descriptive comments.